.github/workflows/release-napi.yml

```diff
@@ -131,6 +131,26 @@ jobs:
           cache: "npm"
           cache-dependency-path: crates/bitwarden-napi/package-lock.json
 
+      - name: Download schemas
+        if: ${{ github.event.inputs.release_type != 'Dry Run' }}
+        uses: bitwarden/gh-actions/download-artifacts@72594be690a4e7bfa87b1402b2aedc75acdbff12
```

Comment:
Add version comment

Reply:
actually, since these are using the bitwarden actions repo, we should just remove the versioning to avoid the renovate PRs. we implicitly trust our own actions...